docs/src/devdocs.md

```diff
@@ -11,7 +13,7 @@ Good practices in developing a code base apply in every Pull Request. The [Good
 3. Add a docstring to your type following the style of the docstrings of other estimators.
 4. If suitable, the estimator may be able to operate based on [`Encoding`](@ref)s. If so, it is preferred to implement an `Encoding` subtype and extend the methods [`encode`](@ref) and [`decode`](@ref). This will allow your probabilities estimator to be used with a larger span of entropy and complexity methods without additional effort. Have a look at the file defining [`SymbolicPermutation`](@ref) for an idea of how this works.
 5. Implement dispatch for [`probabilities_and_outcomes`](@ref) and your probabilities estimator type.
-6. Implement dispatch for [`outcome_space`](@ref) and your probabilities estimator type.
+6. Implement dispatch for [`outcome_space`](@ref) and your probabilities estimator type. The return value of `outcome_space` must be sorted.
```

Comment:
Sorted how? Ascending order? Descending order? 

The point of my previous comment about the sorting was that `sort(outcome_space(est, x)) == outcome_space(est, x)` needs to be `true`. `sort` has a `rev`/`reverse` keyword argument which is either true or false (can't remember) by default). We should be consistent with base Julia, and specify precisely what we mean by "sorted" in this line (point #6).

Reply:
Btw, we should, if we don't already, test this requirement for each estimator explicitly.

Reply:
I think that if someone says "sort" without any specification it means the default behavior of `sort`, which is going from smallest to largest value.

> Btw, we should, if we don't already, test this requirement for each estimator explicitly.

Well at the moment we don't even test the outcome space of the estimators so this should go to #85 .

Reply:
ah actually we do test the outocome space, so I'll add some tests.